### PR TITLE
Cleanup of POCS in preparation of merging PEAS and PACE

### LINKED ISSUE #104: Pre-merge cleanup
Want to preserve history of PEAS and PACE upon merge but first need to do some cleanup.

See also panoptes/PEAS#14

### PR DESCRIPTION
**Pull request by wtgee**
Related to #98 
Closes #104 

Also:
panoptes/PACE#5
panoptes/PEAS#14

### CODE REVIEW

File: pocs/tests/conftest.py [hunk 1]
```diff
@@ -12,12 +12,23 @@
 
 
 def pytest_addoption(parser):
+    parser.addoption("--hardware-test", action="store_true", default=False, help="Test with hardware attached")
```
Comment (jamessynge) [inline pocs/tests/conftest.py]: FWIW, doesn't seem related to the cleanup. Just nitpicking.

Comment (wtgee) [reply pocs/tests/conftest.py]: Yes, good point. It will be need during the merge to make the PEAS tests work (or, rather, skip). I can pull out of here.
